evaluator/src/main/kotlin/RuleSet.kt

```diff
@@ -39,7 +39,7 @@ class RuleSet(val ortResult: OrtResult) {
     val violations = mutableSetOf<RuleViolation>()
 
     /**
-     * The map of all [LicenseFinding]s and associated path excludes by [Identifier].
+     * The map of all [LicenseFindings]s and associated path excludes by [Identifier].
```

Comment:
Nit: `[LicenseFindings]s` should now be `[LicenseFindings]`.

Reply:
I'll address it in the next PR.